cont3xt/vueapp/src/components/services/AuditService.js

```diff
@@ -0,0 +1,72 @@
+import setReqHeaders from '../../../../../common/vueapp/setReqHeaders';
+import Vue from 'vue';
+
+export default {
+  /**
+   * Creates an audit log entry.
+   * @param {Object} audit - The audit data
+   * @returns {Promise} - The promise that either resolves or rejects in error
+   */
+  createAudit (audit) {
+    return new Promise((resolve, reject) => {
+      fetch('api/audit', {
+        method: 'PUT',
+        headers: setReqHeaders({ 'Content-Type': 'application/json' }),
+        body: JSON.stringify(audit)
+      }).then((response) => {
+        return response.json();
+      }).then((response) => {
+        if (response.success) {
+          return resolve(response);
+        } else {
+          return reject(response.text);
+        }
+      });
+    });
+  },
+
+  /**
+   * Fetches the list of audit log entries that a user can view.
+   * @param {Object} query of shape { searchTerm, startMs, stopMs }
+   * @returns {Promise} - The promise that either resolves to an `Audit[]` or rejects in error
+   */
+  getAudits (query) {
+    return new Promise((resolve, reject) => {
+      const options = {
+        method: 'GET',
+        params: query,
+        url: 'api/audits'
+      };
+      Vue.axios(options).then((response) => {
```

Comment:
Can we use fetch here instead of axios. I think that cont3xt has been able to avoid axios altogether (minimizes the bundle size and increases initial load times https://bundlephobia.com/package/axios@0.27.2).